src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculable {
+    protected Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public abstract double getArea();
+
+    public abstract void draw();
```

Comment:
remove, you implemented interface with this method and every type of figure that extends Figure should override this method